### PR TITLE
RTS-1305: Add additional columns for DESCRIBE in README

### PR DESCRIPTION
**Pull request by hazen**

### CODE REVIEW

**Comment by hazen** [comment]
For @gordonguthrie's review

File: README.md [hunk 1]
```diff
@@ -282,15 +282,15 @@ To create a table and see its schema:
...
-|   time    |timestamp| false |     3     |    3    |
-|  weather  | varchar | false |           |         |
-|temperature| double  | true  |           |         |
-+-----------+---------+-------+-----------+---------+
++-----------+---------+-------+-----------+---------+--------+----+
+|  Column   |  Type   |Is Null|Primary Key|Local Key|Interval|Unit|
++-----------+---------+-------+-----------+---------+--------+----+
+| myfamily  | varchar | false |     1     |    1    |        |    |
+| myseries  | varchar | false |     2     |    2    |        |    |
+|   time    |timestamp| false |     3     |    3    |   15   | m  |
```
Comment (hmmr) [inline README.md:290]: I would consider merging the two columns (and the value, which would be "15m"). Horizontal space is precious.

Comment (paegun) [inline README.md:290]: Considering that there is a greater than zero cognitive cost in parsing "15m" into the interval and unit, it is IMHO better UX to have the columns automating this. Also considering product mentions of extending Riak TS to also include Geospatial use cases, so Geospatial functions and units, IMHO the column representation helps as we will also need to pack in the function, ie QUANTUM or GEO.

**Comment by hmmr** [comment]
+1 03cb8d3

**Pull request merged**